bitnami/airflow/templates/config/configmap.yaml

```diff
@@ -149,6 +149,18 @@ data:
             - secretRef:
                 name: {{ .Values.worker.extraEnvVarsSecret }}
             {{- end }}
+            {{- if .Values.extraEnvVarsSecrets }}
+            {{- range .Values.extraEnvVarsSecrets }}
```

Comment:
I would love some feedback as to whether there's a more elegant implementation of this. I'm new to Helm, so this was the best I could do with my current knowledge.